src/Beman/Optional26/tests/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+# cmake-format: off
+# src/Beman/Optional26/tests/CMakeLists.txt -*-makefile-*-
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+# cmake-format: on
+
+# Tests
+add_executable(optional_test "")
```

Comment:
Is the `""` necessary? Also, did you consider adding all the sources as part of the `add_executable` call?

Reply:
@steve-downey Making more complicated code now just in case we need it in the future doesn't have a good set of tradeoffs. Prioritizing readability encourages potential contributors to jump in.